sda-download/api/sda/sda.go

```diff
@@ -450,6 +449,24 @@ func Download(c *gin.Context) {
 	}
 }
 
+var seekStart = func(fileStream io.ReadSeeker, start, end int64) (int64, int64, error) {
```

Comment:
the function `seekStart` does not reflect what is doing very well, maybe change to e.g. `adjustSeekPosition`?

Reply:
good point, fixed in 468848378f87ce5ff92692864ef622a6f7113184